android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -303,7 +328,7 @@ private fun getProductChangeCompletedFunction(onResult: OnResult): (PurchaseDeta
     }
 }
 
-private fun PurchasesError.map(
+fun PurchasesError.map(
```

Comment:
Since this is only used in this module, it can be internal instead of public

Reply:
great catch, I forgot that the default for Kotlin is `public` and assumed it'd be `internal` like swift